### PR TITLE
Implement Kava Executor

### PR DESCRIPTION
**Pull request by denalimarsh**
This PR implements the Kava Executor which enables the deputy to service requests for transfers between Kava and Bnbchain.

A bidirectional transfer guide is located in /executor/kava/guide.md

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -143,18 +155,12 @@ func main() {
 	store.InitTables(db)
 
 	bnbExecutor := bnb.NewExecutor(config.BnbConfig.RpcAddr, types.ChainNetwork(bnbNetwork), config.BnbConfig)
+	kavaExecutor := kava.NewExecutor(config.KavaConfig.RpcAddr, client.ChainNetwork(kavaNetwork), config.KavaConfig)
```
Comment (yutianwu) [inline main.go]: should not just delete eth executor,  we should make it configurable

File: executor/kava/guide.md [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+
+# Transfer bnb from kava to bnbchain
+
+## kava
+
+Create a new Atomic Swap
+
+```bash
+kvcli tx bep3 create ${KAVA_DEPUTY_ADDRESS} ${BNBCHAIN_USER_ADDRESS} ${BNBCHAIN_DEPUTY_ADDRESS} now 1111111bnb 1111111bnb 360 true --from user
+```
```
Comment (HaoyangLiu) [inline executor/kava/guide.md]: I'm confused about this command. On bnbchain command, there are two address: recipient-addr and recipient-other-chain. However, there are three addresses on the above comand? What is the purpose of `BNBCHAIN_DEPUTY_ADDRESS`?

File: main.go (hunk 1 above)
Comment (denalimarsh) [reply main.go]: added in https://github.com/binance-chain/bep3-deputy/pull/1/commits/f5752cb6343bceb0f2ac20040a42ac3ab8b27265, https://github.com/binance-chain/bep3-deputy/pull/1/commits/b0d8d270aee5824a47ef48629438f78621db9a31

File: util/config.go [hunk 3]
```diff
@@ -333,6 +384,10 @@ func ParseConfigFromJson(content string) *Config {
 }
 
 func ParseConfigFromFile(filePath string) *Config {
+	kavaConfig := sdk.GetConfig()
+	app.SetBech32AddressPrefixes(kavaConfig)
```
Comment (yutianwu) [inline util/config.go]: can we move this out

File: util/config.go [hunk 4]
```diff
@@ -6,7 +6,11 @@ import (
 	"io/ioutil"
 	"math/big"
 
+	"github.com/kava-labs/kava/app"
+
 	"github.com/binance-chain/go-sdk/common/types"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
```
Comment (yutianwu) [inline util/config.go]: fix imports here

File: util/config.go (hunk 3 above)
Comment (denalimarsh) [reply util/config.go]: https://github.com/binance-chain/bep3-deputy/pull/1/commits/bf11706ef79fb3546a57768534b75c309ca7697e

File: util/config.go (hunk 4 above)
Comment (denalimarsh) [reply util/config.go]: https://github.com/binance-chain/bep3-deputy/pull/1/commits/bf11706ef79fb3546a57768534b75c309ca7697e

**Pull request merged**